lib/travis_event/handlers.rb

```diff
@@ -0,0 +1,12 @@
+require Rails.root.join('lib/faraday_patch')
```

Comment:
Maybe we should just do this in an intitializer and make it simple (so we don't need lib on the autoload paths and stuff)